homeassistant/components/lifx/light.py

```diff
@@ -151,15 +153,11 @@
 
 def aiolifx():
```

Comment:
I know this is not the purpose of this issue, but this function is unnecessary, could it be removed?

Reply:
Please do it in an other PR. Thanks.